consensus/istanbul/backend/gossip_cache.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 The celo Authors
+// This file is part of the celo library.
+//
+// The celo library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The celo library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the celo library. If not, see <http://www.gnu.org/licenses/>.
+
+package backend
+
+import (
+	lru "github.com/hashicorp/golang-lru"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/consensus/istanbul"
+)
+
+func (sb *Backend) markPeerGossipCache(peerNodeAddr common.Address, payload []byte) {
```

Comment:
Generally I find the names of the functions in here a little confusing, I had to read the contents to get exactly what was happening

Reply:
renamed to the following:

`markPeerGossipCache` -> `markMessageProcessedByPeer`
`checkPeerGossipCache` -> `checkIfMessageProcessedByPeer`
`markSelfGossipCache` -> `markMessageProcessedBySelf`
`checkSelfGossipCache` -> `checkIfMessageProcessedBySelf`